### DISCUSSION TITLE
Appreciation for beepserv-rewrite!

### DISCUSSION THREAD

**Discussion by SteadyStatus21** [thatmarcel/beepserv-rewrite#4] [General]
Hey @thatmarcel! 
I just wanted to open a discussion to show my appreciation for you taking Beeper's original beepserv and rewriting it into a nice looking but basic to understand app! It makes using beepserv much easier and I really appreciate your time spent creating this! 

-Steady :)

**Comment by thatmarcel** [comment]
Thanks a lot for the kind words :)

**Reply by SteadyStatus21** [reply]
Absolutely! The fact that the tweak is an individual app instead of something underneath a toggle makes it so much easier to understand and makes it easier to set it up for the first time, alongside looking for the logs!
